easybuild/easyconfigs/t/triqs/triqs-2.2.1-foss-2019a-Python-2.7.15.eb

```diff
@@ -0,0 +1,66 @@
+easyblock = 'CMakeMake'
+
+name = 'triqs'
```

Comment:
@rokzitko Since `TRIQS` seems to be the official software name (which was also used in #8835, we should stick to that.

If lowercase module names are desired, you should use a lower-case module naming scheme.

Reply:
Also here it would be great if we could from now on use the package naming convention we have for our debian repository as described below. The name would be lower-case `triqs`